wandb/apis/public.py

```diff
@@ -502,6 +504,13 @@ def run(self, path=""):
             self._runs[path] = Run(self.client, entity, project, run)
         return self._runs[path]
 
+    def queued_job(self, path=""):
```

Comment:
is there a particular reason this takes a path instead of just a list of args? seems preferable to avoid string parsing that doesn't need to be string parsing

Reply:
I was mostly trying to mirror the existing API calls